vignettes/fims-path-maturity.Rmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "FIMS Path: Maturity"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{FIMS Path: Maturity}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+This vignette describes the hierarchical structure of FIMS by describing the linkages, or *path* through the FIMS model using maturity as an example. The vignette walks through how input from R is carried into the core FIMS calculations. 
```

Comment:
is "path" a technical concept in programming? I assumed so, but wasn't sure. Maybe if it is, a link would be helpful?

Reply:
This is a somewhat self created term on my part, but I have heard it before within the context of cyclomatic complexity, which is defined as the  number of independent paths through a program. I can't seem to locate any helpful links.